File: 05. Percabangan/Jarak Manhattan/Jarak-Manhattan.cpp
#include <iostream>

int main ()
{
    int x1 = 0, y1 = 0, x2 = 0, y2 = 0;

    std::cin >> x1 >> y1 >> x2 >> y2;

    std::cout << abs((x1 - x2) + (y1 - y2));
    
    return 0;
}

File: 05. Percabangan/Jarak Manhattan/JarakManhattan.java
import java.util.Scanner;

public class JarakManhattan
{
    public static void main(String [] args)
    {
        Scanner scanner = new Scanner (System.in);

        int x1 = scanner.nextInt ();
        int y1 = scanner.nextInt ();
        int x2 = scanner.nextInt ();
        int y2 = scanner.nextInt ();

        scanner.close ();

        System.out.println(Math.abs ((x1 - x2) + (y1 - y2)));
    }
}
